spring-integration-jms/src/test/java/org/springframework/integration/jms/OutboundGatewayFunctionTests.java

```diff
@@ -462,6 +391,18 @@ public Message createMessage(Session session) throws JMSException {
 		scheduler.destroy();
 	}
 
+	private void receiveAndSend(JmsTemplate template) {
+		javax.jms.Message request = template.receive(requestQueue7);
+		final javax.jms.Message jmsReply = request;
+		try {
+			template.send(request.getJMSReplyTo(), (MessageCreator) session -> jmsReply);
+		}
+		catch (JmsException e) {
+		}
+		catch (JMSException e) {
```

Comment:
We can use `multi-catch` here if we already pursuit modern style :smile:  : http://docs.oracle.com/javase/7/docs/technotes/guides/language/catch-multiple.html

![multi-catch](https://cloud.githubusercontent.com/assets/1071590/19704537/f85a84ac-9ad5-11e6-91a6-61c7ed0a9e18.png)


Also pay attention how `MessageCreator` cast is redundant. Plus with lambdas we don't need `final` for passed variables. So, this block can be simplified until this:
```
private void receiveAndSend(JmsTemplate template) {
	javax.jms.Message request = template.receive(requestQueue7);
	try {
		template.send(request.getJMSReplyTo(), session -> request);
	}
	catch (JmsException | JMSException e) {
	}
}
```